testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -1,4 +1,42 @@
 {
     "acceptedApiChanges": [
+        {
```

Comment:
🤔 why do we need this change?

Reply:
These are accepted api changes on lastest master commit we are based on.

Although these are cleaned with master version bump to 8.12 2 days ago, see: https://github.com/gradle/gradle/commit/ab0ab5c8ad26b95c33ef65c86f234cce1998ce6d, but EAP branch is not yet rebased to latest master.